includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -76,90 +76,90 @@ private function getLangWithFallback( $lang ) {
 
 	private $hrefs = [
 		'default' => [
-			'explore-wikis' => '//fandom.wikia.com/explore',
-			'about' => '//www.wikia.com/about',
+			'explore-wikis' => '//www.wikia.com',
```

Comment:
@Grunny we're changing the explore page to N&S feed?

Reply:
We're placing the explore page as the home page of www.wikia.com, so www.wikia.com will serve the content from www.wikia.com/fandom/explore after the switch, rather than the N&S feed.

Reply:
Oh... so what will serve the feed?